### ISSUE TITLE
[MainUI] Allow selecting existing Items to add as points/properties from the Model config page

### ISSUE THREAD

**Issue by rkoshak** [openhab/openhab-webui#433]
When someone is starting from scratch or has already built a semantic model in OH 2 this isn't a big deal. But if someone imports a bunch of Things and Items, or creates a bunch of Things and Items without building the model it becomes awkward adding those already existing Items to the model. One has to leave the model page, find the parent Group Item to add the Items to the right Location or Equipment, then go to the individual Items and edit them to add the semantic tags which takes a ton of clicks.

**Comment by mherwege** [comment]
Would be solved with model drag & drop: https://github.com/openhab/openhab-webui/pull/2970

**Comment by rkoshak** [comment]
I don't think so. I'm referring to when  you choose "Add equipment to thing" or "Add points to model" from the Thing's Channels page.

![Image](https://github.com/user-attachments/assets/e5b87997-afc0-4e62-87d7-4777c1d06233)

Which leads to the form where you select the Location, select/create the Location, then choose the Channels to link to *new* Items.

![Image](https://github.com/user-attachments/assets/10f34c7a-b293-4732-9e77-268295184265)

I want to link the Channel to an existing Item from here instead of only having the option to create a new Item. 

#2970 helps with different use cases but not with this initial linking step.

**Comment by mherwege** [comment]
OK, sorry, I misunderstood.

**Comment by mherwege** [comment]
> One has to leave the model page, find the parent Group Item to add the Items to the right Location or Equipment, then go to the individual Items and edit them to add the semantic tags which takes a ton of clicks.

Model drag/drop would allow you to do it from the model tree. You just drag the item to the right location or equipment. No need to add semantic tags manually.

**Comment by rkoshak** [comment]
> Model drag/drop would allow you to do it from the model tree. You just drag the item to the right location or equipment. No need to add semantic tags manually.

True, but that's not the main focus of this issue. Though the way I worded the original issue I agree it kind of sounds like it. The whole issue of situating the Items into the model are indeed solved by the drag and drop PR. But it still would be great if we could select existing Items and not just create new Items from this form above and beyond the whole model discussion.

**Comment by mherwege** [comment]
@rkoshak Would it be OK to be allowed to pick (one of) the existing item(s) already linked to the channel selected? Or would you want to also allow linking existing items, not linked to a channel or to another channel potententially on another thing? I can see the first to make a lot of sense. I am more doubtful about the second.

**Comment by mherwege** [comment]
But isn't that already possible with the 'Add Link to Item' functionality on the previous screen? You see all the channels, and each of them has an 'Add Link to Item' link.
I am a bit afraid to introduce this general approach behind the 'Add Equipment to Model' and 'Add Points to Model' links. These are purely there for creating the model. I can see value into having some update functionality when you don't start from scratch (i.e. make an existing item linke to a channel part of the model and make the thing an equipment with all its existing channel linked items), but that should also guard to some extent the consistency of the model.

I was thinking about allowing using existing linked items for the model, and making sure the semantic meta data is set according to the option chosen ('Equipment to Model', 'Points to Model') for these existing items.

If its managing channel item links outside of the model, I don't think this is the right place. Having it there will only create a risk of making the model inconsistent and may confuse more than it helps.
